extension/README.md

```diff
@@ -0,0 +1,35 @@
+# arcs chrome extension
+
+The extension does two things:
+
+1) Synchronizes metadata from the browser to the shared Arcs system.
+1) Provides an entry point to Arcs.
+
+The first point is worth re-emphasizing. **The extension sends metadata from
```

Comment:
An alternative would be that the extension shows the data it has extracted, and allows the user to flush all of it or a subset after inspection.

Reply:
Good idea - I made #442 for that.